artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/OperationContextImpl.java

```diff
@@ -104,6 +107,29 @@ public static void setContext(final OperationContext context) {
 
    private final Executor executor;
 
+   public static final String MAX_STORE_OPERATION_TRACKERS_PROP = "artemis.maxStoreOperationTrackers";
+
+   private static int maxStoreOperationTrackers = -1;
```

Comment:
- I think this should be final 
and it should be initialized with a static initializer, or simply use the System.getProperty straight away.

{
}

- Also the property name.  and the pattern used here.

RefCountMessage used a different pattern. I think we should stick to the same pattern used in there.


ARTEMIS_


and maxStoreOperationsTrackers.. it doesn't not really translate to anything unless I read the code and understand what is doing.

I'm not sure what better name we could use (I'm terrible with names). but what about:

ARTEMIS_OPCONTEXT_TRACKER_NUMBER?

And some javadoc of what it does?

Reply:
What about `ARTEMIS_OPCONTEXT_MAX_DEBUG_TRACKERS`/`maxDebugTrackers` ?

Reply:
done